python/lsst/analysis/ap/plotImageSubtractionCutouts.py

```diff
@@ -908,11 +922,20 @@ def run_cutouts(args):
         sources = cutouts.run(data, butler, njobs=args.jobs)
     else:
         sources = []
+        all_data = []
         count = len_sources(apdb_query)
         for i, data in enumerate(getter):
+            all_data.append(data)
```

Comment:
This will use a ton of memory; you might consider either only saving the relevant columns.